packages/patternfly-4/react-table/src/components/Table/Table.md

```diff
@@ -290,6 +290,58 @@ class SortableTable extends React.Component {
 }
 ```
 
+```js title=Sortable-with-wrapping-headers
+import React from 'react';
+import {
+  Table,
+  TableHeader,
+  TableBody,
+  sortable,
+  SortByDirection,
+  wrappable,
+} from '@patternfly/react-table';
+
+class SortableTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      columns: [
+        {title: 'This is a really long table header that goes on for a long time 1.', transforms: [sortable, wrappable]},
```

Comment:
It's a little hard to see what's going on because of the line length of the titles. Not sure if our linters automatically format like this, otherwise we might be able to make it more clear by adding some simple line breaks. Makes it take up a little more vertical space, but also makes it really easy to see that this functionality is achieved by composing the sort/wrap transforms.

<img width="600" alt="Screen Shot 2020-02-12 at 10 09 02 AM" src="https://user-images.githubusercontent.com/5942899/74348350-8a51f100-4d80-11ea-94f4-eb99a6dec668.png">

VS

<img width="600" alt="Screen Shot 2020-02-12 at 10 11 00 AM" src="https://user-images.githubusercontent.com/5942899/74348352-8aea8780-4d80-11ea-8e71-7babd5553c30.png">

Reply:
I agree that looks more readable. :)